address issues in DOC-3935 (#411)
redis/docs: content/develop/interact/search-and-query/advanced-concepts/tags.md

FT.CREATE ... SCHEMA ... {field_name} TAG [SEPARATOR {sep}] [CASESENSITIVE]
```

> SEPARATOR defaults to a comma (`,`) for HASH only, and can be any printable ASCII character. For JSON, there is no default separator and you must declare one explicitly if needed.  

Reviewer: ```suggestion
For hashes, SEPARATOR can be any printable ASCII character; the default is a comma (`,`). For JSON, there is no default separator; you must declare one explicitly if needed.
```
